components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -19,31 +20,44 @@ interface InputBoxProps {
   onHandleAgreeClick: Function
 }
 
+const MAX_INPUT_CHAR = 2000
+const CHAR_LIMIT_THRESHOLD = MAX_INPUT_CHAR * 0.80
```

Comment:
nit: "Char Limit Threshold" seems to me to be another way to express "Max input char" (i.e. the threshold of the limit of characters). But it seems like it's being used as the threshold to display the limit. Since you've put this at the top of the file, I assume you want it to be easily understandable and modified, so I wonder if there's a clearer name, e.g. `INPUT_CHAR_COUNT_DISPLAY_THRESHOLD` or `INPUT_CHAR_LIMIT_APPROACHING_THRESHOLD`